xbmc/pvr/PVRDatabase.cpp

```diff
@@ -45,7 +45,7 @@ void CPVRDatabase::CreateTables()
 {
   CLog::Log(LOGINFO, "PVR - %s - creating tables", __FUNCTION__);
 
-  CLog::Log(LOGDEBUG, "PVR - %s - creating table 'channels'", __FUNCTION__);
+  CLog::Log(LOGDEBUG, "CPVRDatabase - %s - creating table 'channels'", __FUNCTION__);
```

Comment:
the component is called "PVR" and there's only one db in there. if you used grep already, how can you not find the correct line that produced this?